src/engine/strat_engine/thindev.rs

```diff
@@ -2,23 +2,31 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-
 use devicemapper::{DM, DevId, DeviceInfo, DmFlags};
 use engine::{EngineError, EngineResult, ErrorEnum};
 use engine::strat_engine::thinpooldev::ThinPoolDev;
+
+use std::fmt;
 use std::path::Path;
 use std::path::PathBuf;
 use std::process::Command;
+
 use types::Sectors;
 
 #[derive(Clone)]
 pub struct ThinDev {
-    name: String,
+    pub name: String,
```

Comment:
Are you sure any of this has to be public? Within a module, even private fields are public.